src/vector/v.maxent.swd/v.maxent.swd.html

```diff
@@ -1,54 +1,80 @@
 <h2>DESCRIPTION</h2>
 
-<em>v.maxent.swd</em> Produce a set of SWD files as input to MaxEnt 3.3.3k
-(version 3.4.1 untested, but should work as well), based on a point vector
-layer(s) of species distribution(s), a point layer with background points and
-user-defined raster layers. The user can also let the function create the
-background point file. (respecting region and mask, if set).
-
-<p>The SWD file format is a simple CSV-like file file format as
-described in Elith et al. 2011. The first three fields provide the
-species name, x-coordinate and y-coordinate, while subsequent fields
-contain the values of the user-selected environmental parameters.
-Note that the coordinates are not used by the MaxEnt program if all
-environmental data is given in the SWD format file.
-
-<p>The user can provide one or more species point files, which will be
-combined in one SWD file. The map names of both the species point
-layers and the environmental parameters can be replaced by alias
-names, which will be used by MaxEnt.
+<em>v.maxent.swd</em> Produce a set of SWD files as input to MaxEnt 
+3.3.3k+, based on a point vector layer(s) of species distribution(s), a 
+point layer with background points and user-defined raster layers. The 
+user can also let the function create the background point file. 
+(respecting region and mask, if set).
+
+<p>
+The SWD file format is a simple CSV-like file file format as described 
+in Elith et al. 2011. The first three fields provide the species name, 
+x-coordinate and y-coordinate, while subsequent fields contain the 
+values of the user-selected environmental parameters. Note that the 
+coordinates are not used by the MaxEnt program if all environmental 
+data is given in the SWD format file.
+
+<p>
+The user can provide one or more species point files, which will be 
+combined in one SWD file. The map names of both the species point 
+layers and the environmental parameters can be replaced by alias names, 
+which will be used by MaxEnt.
+
+<p>
+If alias names are used , a CSV file (alias_file) can be created with 
+alias names in the first column and map names in the second column, 
+separated by comma, without header. This can be used as input in 
+r.maxent.lambdas (but note that this addon does not seem to work with 
+outputs from Maxent 3.4+).
```

Comment:
Please open an issue / PR. From a brief look at the change log of Maxent I guess the issue is the transformation function!?

Reply:
@ninsbl I was planning, but first wanted to check if it is only the transformation. A quick test yielded a different result from Maxent using the logistic transformation. But I need to check that more carefully.